frontend/lib/ui/artefact_page/test_event_log_expandable.dart

```diff
@@ -18,54 +18,57 @@ class TestEventLogExpandable extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    return ExpansionTile(
-      controlAffinity: ListTileControlAffinity.leading,
-      childrenPadding: const EdgeInsets.only(left: Spacing.level4),
-      shape: const Border(),
+    return Expandable(
       title: const Text('Event Log'),
       initiallyExpanded: initiallyExpanded,
       children: <Widget>[
-        SingleChildScrollView(
-          scrollDirection: Axis.horizontal,
-          child: DataTable(
-            columns: const <DataColumn>[
-              DataColumn(
-                label: Expanded(
-                  child: Text(
-                    'Event Name',
-                    style: TextStyle(fontStyle: FontStyle.italic),
-                  ),
+        DataTable(
+          columns: const <DataColumn>[
+            DataColumn(
+              label: Expanded(
+                child: Text(
+                  'Event Name',
+                  style: TextStyle(fontStyle: FontStyle.italic),
                 ),
               ),
-              DataColumn(
-                label: Expanded(
-                  child: Text(
-                    'Timestamp',
-                    style: TextStyle(fontStyle: FontStyle.italic),
-                  ),
+            ),
```

Comment:
It seems that here we have some formatting changes, should we rebase to main to use the formatter you added in the previous PR?

Reply:
There's no formatting changes here (already merged with main). But I've removed the `SingleChildScrollView`